pandas_summary/__init__.py

```diff
@@ -252,7 +252,23 @@ def _get_column_summary(self, column):
             return self._get_date_summary(column)
         if column_type == self.TYPE_CONSTANT:
             return self._get_constant_summary(column)
-
+        
+    @property
+    def get_constants(self):
```

Comment:
I think it would make sense to either leave the functions with `get_...` and remove the `@property`, or remove the `get_` and keep them as properties.

What do you think?

Reply:
Second idea makes more sense to me.
It's more easy to read
```
dfs.constants # [A, B]
dfs.bools # [C, D, E]
```